doc/GUIDE.md

```diff
@@ -103,94 +99,33 @@ michael@d30748af6d3d:~$ stack new helloworld new-template
 
 For this first stack command, there's quite a bit of initial setup it needs to
 do (such as downloading the list of packages available upstream), so you'll see
-a lot of output. Though your exact results may vary, below is an example of the
-sort of output you will see. Over the course of this guide a lot of the content
-will begin to make more sense:
+a lot of output. Over the course of this guide a lot of the content will begin
+to make more sense:
```

Comment:
Probably a period instead of a colon here, since the output it was pointing to is removed now.

Reply:
Oh, good catch.